src/staging/staging_main.py

```diff
@@ -152,9 +152,9 @@ def run_staging(
         StagingMainLogger.info("Loading of Manual Outlier File skipped")
 
     # Load the PG mapper
-    mapper_path = paths["mapper_path"]
-    check_file_exists(mapper_path)
-    mapper = read_csv(mapper_path)
+    pg_mapper = paths["mapper_path"]
```

Comment:
Sorry could you also change the name of the argument in the `dev_config` pls. Then the following suggestion should work (just for consistency!)

```suggestion
    pg_mapper = paths["pg_mapper_path"]
```

Reply:
mapper = read_csv(mapper_path)
  what do you think about "mapper" need to be change as "pg_mapper",  or just leave it as mapper

Reply:
Yeh I think change "mapper" to "pg_mapper"